### PR TITLE
feat: [IAC-621]: Add iacm terraform plugin

### PR DESCRIPTION
**Pull request by willd0yle**
### Summary
Add IACMTerraformPlugin pipeline step:

#### Screenshots
Screenshot 2023-05-25 at 13 45 51
Screenshot 2023-05-25 at 13 45 28
NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by nassergonzalez — APPROVED** [review]

**Review by litvinovstas — APPROVED** [review]

File: src/modules/31-filestore/components/ActivityLog/ActivityLogFileStore.module.scss.d.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /* eslint-disable */
-/*
- * Copyright 2023 Harness Inc. All rights reserved.
+/**
+ * Copyright 2021 Harness Inc. All rights reserved.
```
Comment (nassergonzalez) [inline src/modules/31-filestore/components/ActivityLog/ActivityLogFileStore.module.scss.d.ts:3]: maybe revert this change in the year xD

**Review by nassergonzalez — APPROVED** [review]

File: src/modules/31-filestore/components/ActivityLog/ActivityLogFileStore.module.scss.d.ts (hunk 1 above)
Comment (willd0yle) [reply src/modules/31-filestore/components/ActivityLog/ActivityLogFileStore.module.scss.d.ts:3]: not sure why it happened?

Comment (knagurski) [reply src/modules/31-filestore/components/ActivityLog/ActivityLogFileStore.module.scss.d.ts:3]: It's caused by a plugin. Unfortunately it'll revert to 2021 every time the plugin runs, which is every time that CSS file is evaluated.
